testnet/tools/nns-tools/submit-mainnet-insert-upgrade-path-proposal.sh

```diff
@@ -63,15 +65,19 @@ submit_insert_upgrade_path_proposal_mainnet() {
     )
 
     if [ "${TARGET_SNS_GOVERNANCE_CANISTER}" == "All" ]; then
-        cmd+=("--force-upgrade-main-upgrade-path true")
+        cmd+=("--force-upgrade-main-upgrade-path=true")
     else
-        cmd+=("--sns-governance-canister-id ${TARGET_SNS_GOVERNANCE_CANISTER}")
+        cmd+=("--sns-governance-canister-id=${TARGET_SNS_GOVERNANCE_CANISTER}")
```

Comment:
I guess this addresses

> clap thinks the entire --sns-governance-canister-id xxx-xxx as the argument name

Not sure what you meant there though. Can you explain (in the PR description) what clap does when space is used instead of =?

I actually like = better than space (it clearly indicates that things are closely related to one another). I'm justt not understanding what clap's confusion is.